### PR TITLE
Users info UI

### PR DESCRIPTION
**Pull request by 0xAnon0602**
Added the User Information with [RIPE API](https://stat.ripe.net/) and [IYP data](https://iyp.iijlab.net/) as discussed in #758 

image

### CODE REVIEW

**Comment by dpgiakatos** [comment]
Some improvements for the UI, before I review your PR. Please, add connection information below the Global Report button. Optimize the card layout for compactness by removing unnecessary space. Also, include a line to separate the button from the card.

**Comment by 0xAnon0602** [comment]
Got it 
Will work on the changes you suggested

**Comment by dpgiakatos** [comment]
Once you change the UI please upload a screenshot here.

**Comment by 0xAnon0602** [comment]
@dpgiakatos 
You mean by moving that card in the way I showed in the image right ?
image

**Comment by dpgiakatos** [comment]
Do we have any progress on this PR?

**Comment by 0xAnon0602** [comment]
Hey sorry for the delay was busy during the weekend
Will probably push the new changes by the EOD

**Comment by 0xAnon0602** [comment]
@dpgiakatos 
I was able to do the changes as you said 
Let me know if any more changes are required in the UI

image

**Comment by dpgiakatos** [comment]
The card should have the same width with the button and the title. Also, as I have mentioned before, you should have a line, which will separate the button and the card. Furthermore, the card should follow the buttons style; the corners should not be round. The same goes with the background, make it transparent and the font color should be white (like the button and the title). Remove the icon in cards title. The card’s border should be white.

**Comment by 0xAnon0602** [comment]
@dpgiakatos 

Is this what you were looking for ?

image

**Comment by dpgiakatos** [comment]
It's almost there, but some more modifications should be made. The corners are still round, and the card's width is slightly bigger than the button. So, let's remove the card's radius and ensure that the card, button, and title have the same width. Additionally, remove the underline from 'Your Connections', make it uppercase, and add a border to the card's title (see example in the attachment). Finally, the card's text should be like the example I have attached

image

**Comment by 0xAnon0602** [comment]
@dpgiakatos 
Added the changes you asked 
Let me know if any further change is required )

image

File: src/components/UserInfo.vue
Comment (dpgiakatos) [inline src/components/UserInfo.vue:1]: When the screen shrinks, the component disappears, which it should not.

File: src/views/Home.vue
Comment (dpgiakatos) [inline src/views/Home.vue:1]: The recent changes have altered the current UI layout, which needs to be fixed. The UI layout should remain consistent and not change unexpectedly. Please revert the changes to restore the UI to its previous state as in the dev branch. Additionally, ensure that the addition of the UserInfo component does not affect the layout of the landing page.

---PR 765---
image
---
---Dev branch---
image
---

**Review by dpgiakatos — CHANGES_REQUESTED** [review]
This pull request introduces a new feature, which is a nice addition. However, before approving it, there are several issues that need to be addressed. The contributor should focus on resolving the issues mentioned in this review before proceeding further.

**Comment by 0xAnon0602** [comment]
@dpgiakatos 

Did all the changes as you asked me and this is the updated UI

image

Let me know if I should open a new PR if this one is conflicting with the `dev` branch

**Comment by 0xAnon0602** [comment]
@dpgiakatos 
I've linked the final PR in #769

**Pull request closed without merging**